api/controllers/user.go

```diff
@@ -0,0 +1,127 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"net/mail"
+	"os"
+
+	"github.com/PedroChaparro/loomies-backend/interfaces"
+	"github.com/PedroChaparro/loomies-backend/models"
+	"github.com/gin-gonic/gin"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"golang.org/x/crypto/bcrypt"
+)
+
+const uri = "mongodb://root:development@localhost:27017/"
```

Comment:
This is not needed. Controllers should use the model functions to interact with the database.

Reply:
I was testing because I had a problem with the database and I forgot to delete that